README.md

```diff
@@ -37,9 +37,12 @@ The process of testing changes to Twoliter in Bottlerocket is as follows:
 - Commit your changes. This can be either a local commit or a commit on a git fork.
 - In the Bottlerocket git repo
   - Remove the existing Twoliter binary if it exists.
+  - Ensure the [nightly rust toolchain] is installed and enabled.
```

Comment:
These instructions seem dated to me. Running `cargo make` in the Bottlerocket repo won't know how to locate the test `twoliter` to build. Can you clarify the workflow you followed?

Typically I `cd` to the `twoliter` repo and do either `make build` or `cargo build --release --locked` and copy the `twoliter` binary into `bottlerocket/tools/twoliter/` in the Bottlerocket repo.


I'm OK with instructions to use the nightly compiler, but it should already be handled by this config file:

https://github.com/bottlerocket-os/twoliter/blob/develop/rust-toolchain.toml

Reply:
Yes, I followed [this workflow ](https://github.com/bottlerocket-os/twoliter?tab=readme-ov-file#testing-twoliter-changes-in-the-bottlerocket-repo) when I encountered this nightly error.

This was my build cmd:
```
cargo make -e TWOLITER_REPO=https://github.com/koooosh/twoliter-fork.git \
    -e TWOLITER_VERSION=fef3dce \
    -e TWOLITER_ALLOW_SOURCE_INSTALL=true \
    -e TWOLITER_ALLOW_BINARY_INSTALL=false \
    -e TWOLITER_SKIP_VERSION_CHECK=true \
    -e BUILDSYS_VARIANT=aws-k8s-1.31
```

Reply:
Ah, I forgot that that interface existed!